Exec/mhd_tests/Detonation/bc_fill_nd.F90

```diff
@@ -0,0 +1,125 @@
+module bc_fill_module
```

Comment:
I think we shouldn't do special boundary conditions, because I don't think these handle the magnetic fields.  So let's just use the normal BCs to start.